lms/djangoapps/teams/templates/teams/teams.html

```diff
@@ -34,8 +34,8 @@
 <%static:require_module module_name="teams/js/teams_tab_factory" class_name="TeamsTabFactory">
     TeamsTabFactory({
         courseID: '${ unicode(course.id) }',
-        topics: ${ json.dumps(topics, cls=EscapedEdxJSONEncoder) },
-        userInfo: ${ json.dumps(user_info, cls=EscapedEdxJSONEncoder) },
+        topics: ${ escape_json_dumps(topics) | n },
+        userInfo: ${ escape_json_dumps(user_info) | n },
         topicUrl: '${ topic_url }',
```

Comment:
Yes (for topicUrl).  And we spoke and agreed to not expand scope.  Additional variables used in javascript will be escaped as part of the "h" as default filter stories.